packages/authentication-oauth/test/express.test.ts

```diff
@@ -40,12 +48,4 @@ describe('@feathersjs/authentication-oauth/express', () => {
       delete app.get('authentication').oauth.redirect;
     }
   });
-
-  it('oauth/test/authenticate with error', async () => {
```

Comment:
Not sure what to do with that test. The thrown error was only implemented in the mocked `getProfile` function. Not sure what that was supposed to test in the first place.

Since `getProfile` doesn't exist anymore, I just removed the test for now.